MetadataMigration/src/main/java/org/opensearch/migrations/commands/Migrate.java

```diff
@@ -31,9 +31,9 @@ public MigrateResult execute(RootMetadataMigrationContext context) {
         } catch (ParameterException pe) {
             log.atError().setCause(pe).setMessage("Invalid parameter").log();
             migrateResult
-                .exitCode(INVALID_PARAMETER_CODE)
-                .errorMessage("Invalid parameter: " + pe.getMessage())
-                .build();
+                    .exitCode(INVALID_PARAMETER_CODE)
```

Comment:
Ugh, this is my IDE being out of sync with the usual settings. I'll go through and clean up the indents.

Reply:
I had a PR once upon a time for an intellij formatter [1], there is a VS code in the project too.

- https://github.com/opensearch-project/opensearch-migrations/pull/846